API/Services/ForecastDataRetrievalService.cs

```diff
@@ -18,20 +18,20 @@ public interface IForecastDataRetrievalService
 
     public class ForecastDataRetrievalService : IForecastDataRetrievalService
     {
-        private HttpClient HttpClient { get; init; }
+        private HttpClient HttpClient { get; }
         public ForecastDataRetrievalService()
         {
-            this.HttpClient = new HttpClient();
-            this.HttpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
-            this.HttpClient.DefaultRequestHeaders.UserAgent.ParseAdd(
+            HttpClient = new HttpClient();
+            HttpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            HttpClient.DefaultRequestHeaders.UserAgent.ParseAdd(
                 "Mozilla/5.0 (Macintosh; Intel Mac OS X x.y; rv:42.0) Gecko/20100101 Firefox/42.0");
```

Comment:
This has been changed in `master`. Perhaps merge/rebase master into this branch?

Reply:
Will do! 👍